src/allmydata/dirnode.py

```diff
@@ -331,8 +331,11 @@ def _decrypt_rwcapdata(self, encwrcap):
         salt = encwrcap[:16]
         crypttext = encwrcap[16:-32]
         key = hashutil.mutable_rwcap_key_hash(salt, self._node.get_writekey())
-        cryptor = AES(key)
-        plaintext = cryptor.process(crypttext)
+        # XXX uhm, so maybe this is confusing even if it's what the
+        # original code was doing; worth making a aes.decrypt_data
+        # just to be more clear?
+        encryptor = aes.create_encryptor(key)
+        plaintext = aes.encrypt_data(encryptor, crypttext)
```

Comment:
I think it probably is worth doing that, yea.  It also gives us a natural place to hang an explanation of why re-encrypting the ciphertext gives us the plaintext, if we feel like we want to explain that.

Reply:
Ah, I already made "decrypt" versions .. just didn't use them here. I guess further: is it "too weird" that they're just aliases?